projects/storefrontlib/src/lib/ui/pages/default-routing-config.ts

```diff
@@ -8,7 +8,10 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   register: { paths: ['register'] },
   resetPassword: { paths: ['login/pw/change'] },
   forgotPassword: { paths: ['forgot-password'] },
-  checkout: { paths: ['checkout'] },
```

Comment:
Should we remove this route? Other components might use it? If those components should link to checkout this one should be used.

Reply:
At least 2 components are using this checkout route, so let's keep it.

Reply:
I reverted that change.